salesagility/SuiteCRM
# Fix 'On calendar scheduling bar - no user icon'

## Description
Added missing icons to Suite P for the meetings scheduler.

## Motivation and Context
Suite P was displaying broken icons on the meetings page

## How To Test This
Set your theme to Suite P then create or edit a meeting. If you go to add a user/contact/lead to the meeting there should be an icon next to the user/contact/lead name. Previously this was broken.

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)

### Final checklist


- [ x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ x] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.
